test/unit/amplitude/test_yaml_helicity.py

```diff
@@ -21,10 +21,11 @@
 
 @pytest.fixture(scope="module")
 def amplitude_generator():
-    initial_state = [("J/psi", [-1, 1])]
-    final_state = [("gamma", [-1, 1]), ("pi0", [0]), ("pi0", [0])]
-
-    stm = StateTransitionManager(initial_state, final_state, ["f"])
+    stm = StateTransitionManager(
+        initial_state=[("J/psi", [-1, 1])],
+        final_state=["gamma", "pi0", "pi0"],
+        allowed_intermediate_particles=["f"],
+    )
```

Comment:
Can you avoid making such "unrelated" changes in the PR. I know its boy scout rule, but it just confuses git. When files are being moved and one line is edited, then it marks it as deleted and creates a new file.

Reply:
Review commit by commit

Reply:
I did. I just dont like the way github shows moved files as deleted and new files

Reply:
It went wrong for the `__init__.py` files because there the word `tests` changed into `test`. But I'll  keep it in mind for next time.